examples/tic-tac-toe/client.js

```diff
@@ -13,22 +13,20 @@ function initUI(cells, playerIds, players, yourPlayerId) {
   })
 
   playerContainers = playerIds.map((playerId, index) => {
+    const { displayName, avatarUrl } = Rune.getPlayerInfo(playerId)
```

Comment:
Could be wrong, but I don't think this will work for spectator - undefined playerId still returns undefined.